tap_outreach/discover.py

```diff
@@ -28,9 +28,11 @@ def get_schemas():
             singer.resolve_schema_references(schema, refs)
 
         replication = STREAM_CONFIGS[stream_name]['replication']
+        replication_keys = STREAM_CONFIGS[stream_name].get('filter_field', None)
```

Comment:
Is `replication_keys` a list or a single value? If single value can we change it to `replication_key`?

Reply:
Fixed in latest commit.